src/main/java/connectDB/DriverDB.java

```diff
@@ -0,0 +1,17 @@
+package connectDB;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DriverDB {
+    public static Connection initializeDatabase() throws SQLException, ClassNotFoundException {
+        String dbDriver = "org.postgresql.Driver";
+        String URL = "jdbc:postgresql://localhost:5432/userdb";
+        String USERNAME = "postgres";
+        String PASSWORD = "qwerty";;
+
+        Class.forName(dbDriver);
```

Comment:
We should move it to a separate method not to load this class every time, we need to get a connection